plugins/generator.md

```diff
@@ -0,0 +1,91 @@
+## Generators
+
+### Plugin generator
+
+Use the plugin generator to create a ManageIQ plugin.
+
+For example, to create a plugin called ManageIQ::MyPlugin, do the following
```

Comment:
Do we have an explanation of when to use the ManageIQ namespace or not?  Maybe it doesn't belong here but that was my first thought...

Reply:
There is no need to use the ManageIQ namespace, so I didn't mention it.  I'm not sure how I would express that.